packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -461,6 +476,15 @@ if (pg_info.database) {
         const policies = await MDSDBPostgres.findPoliciesByGeographyID(LAGeography.geography_id)
         assert.deepEqual(policies[0].policy_id, POLICY3_JSON.policy_id)
       })
+
+      it('throws if both get_published and get_unpublished are true for bulk geo reads', async () => {
+        await assert.rejects(
```

Comment:
I'm going to have to remember this pattern :D

Reply:
You can actually specify the full error object itself rather than just the name with `assert.throws`, check out our tests in `mds-schema-validators`